[2021-07-08 12:15:33,010] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.dump_user_log 2021-07-08T12:15:24.497141+00:00 [queued]>
[2021-07-08 12:15:33,034] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.dump_user_log 2021-07-08T12:15:24.497141+00:00 [queued]>
[2021-07-08 12:15:33,034] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 12:15:33,035] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-08 12:15:33,035] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 12:15:33,046] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): dump_user_log> on 2021-07-08T12:15:24.497141+00:00
[2021-07-08 12:15:33,053] {standard_task_runner.py:52} INFO - Started process 14012 to run task
[2021-07-08 12:15:33,059] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'bluepi_dag', 'dump_user_log', '2021-07-08T12:15:24.497141+00:00', '--job-id', '85', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bluepi_dag.py', '--cfg-path', '/tmp/tmpshzae_vk', '--error-file', '/tmp/tmpxpudngzf']
[2021-07-08 12:15:33,064] {standard_task_runner.py:77} INFO - Job 85: Subtask dump_user_log
[2021-07-08 12:15:33,124] {logging_mixin.py:104} INFO - Running <TaskInstance: bluepi_dag.dump_user_log 2021-07-08T12:15:24.497141+00:00 [running]> on host 3cf0296804bd
[2021-07-08 12:15:33,186] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=natchanon
AIRFLOW_CTX_DAG_ID=bluepi_dag
AIRFLOW_CTX_TASK_ID=dump_user_log
AIRFLOW_CTX_EXECUTION_DATE=2021-07-08T12:15:24.497141+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-08T12:15:24.497141+00:00
[2021-07-08 12:15:33,201] {base.py:78} INFO - Using connection to: id: my_postgres_conn. Host: 35.247.174.171, Port: 5432, Schema: postgres, Login: exam, Password: ***, extra: {}
[2021-07-08 12:15:34,894] {python.py:151} INFO - Done. Returned value was: None
[2021-07-08 12:15:34,910] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=bluepi_dag, task_id=dump_user_log, execution_date=20210708T121524, start_date=20210708T121533, end_date=20210708T121534
[2021-07-08 12:15:34,942] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-08 12:15:34,963] {local_task_job.py:151} INFO - Task exited with return code 0
